### DISCUSSION TITLE
Is it possible to cross-load a Nitrokey 3 firmware onto a Solo2?

### DISCUSSION THREAD

**Discussion by VA1DER** [solokeys/solo2#178] [Q&A]
Is it possible to cross-load a Nitrokey 3 firmware onto a Solo2 hacker?

**Comment by msiqx9** [comment]
The answer is [No](https://github.com/solokeys/solo2/discussions/174#discussioncomment-8021829).

**Comment by VA1DER** [comment]
The answer is [probably yes](https://support.nitrokey.com/t/is-it-possible-to-cross-load-nitrokey-3-firmware-to-solo2/5617), with some work.

**Reply by msiqx9** [reply]
> The answer is probably yes, with some work.

How are you gettng on with this project?

**Reply by msiqx9** [reply]
There seems to be some code [activity](https://github.com/solokeys/solo2/commit/bf0e51fc44daedb7b8f9edd6fb9e27b00d2fd319) yesterday in this area.
